File: official/docs/java/v5/child-users/delete.java
package users;

import com.easypost.EasyPost;
import com.easypost.exception.EasyPostException;
import com.easypost.model.User;

public class Delete {
    public static void main(String[] args) throws EasyPostException {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        User user = User.retrieve("user_...");

        user.delete();

        System.out.println(user);
    }
}

File: official/docs/csharp/v4/child-users/delete.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client("EASYPOST_API_KEY");

            EasyPost.Models.API.User user = await client.User.Retrieve("user_...");

            bool success = await user.Delete();

            Console.WriteLine(success.ToString());
        }
    }
}
